app/controllers/api/v1/resources_controller.rb

```diff
@@ -1,9 +1,25 @@
-class Api::V1::ResourcesController < ApplicationController
+require 'pry'
+class Api::V1::ResourcesController < ApiController
```

Comment:
require pry isnt needed. its auto loaded in all controllers and models and views by default